Terminal.Gui/Views/Toplevel.cs

```diff
@@ -221,29 +221,28 @@ void SetInitialProperties ()
 
 			// Default keybindings for this view
 			AddKeyBinding (Application.QuitKey, Command.QuitToplevel);
-			AddKeyBinding (Key.Z | Key.CtrlMask, Command.Suspend);
-
-			AddKeyBinding (Key.Tab, Command.NextView);
 
 			AddKeyBinding (Key.CursorRight, Command.NextView);
-			AddKeyBinding (Key.F | Key.CtrlMask, Command.NextView);
-
 			AddKeyBinding (Key.CursorDown, Command.NextView);
-			AddKeyBinding (Key.I | Key.CtrlMask, Command.NextView); // Unix
-
-			AddKeyBinding (Key.Tab | Key.ShiftMask, Command.PreviousView);
 			AddKeyBinding (Key.CursorLeft, Command.PreviousView);
 			AddKeyBinding (Key.CursorUp, Command.PreviousView);
-			AddKeyBinding (Key.B | Key.CtrlMask, Command.PreviousView);
 
+			AddKeyBinding (Key.Tab, Command.NextView);
+			AddKeyBinding (Key.Tab | Key.ShiftMask, Command.PreviousView);
 			AddKeyBinding (Key.Tab | Key.CtrlMask, Command.NextViewOrTop);
-			AddKeyBinding (Application.AlternateForwardKey, Command.NextViewOrTop); // Needed on Unix
-
 			AddKeyBinding (Key.Tab | Key.ShiftMask | Key.CtrlMask, Command.PreviousViewOrTop);
-			AddKeyBinding (Application.AlternateBackwardKey, Command.PreviousViewOrTop); // Needed on Unix
 
-			AddKeyBinding (Key.L | Key.CtrlMask, Command.Refresh);
+			AddKeyBinding (Key.F5, Command.Refresh);
+			AddKeyBinding (Application.AlternateForwardKey, Command.NextViewOrTop); // Needed on Unix
+			AddKeyBinding (Application.AlternateBackwardKey, Command.PreviousViewOrTop); // Needed on Unix
 
+#if UNIX_KEY_BINDINGS
```

Comment:
I think it's better to use the a platform condition, that if it's running `Linux` or `macOS` these key bindings can be used, otherwise are ignored. Don't you think so?

Reply:
This is temporary. See https://github.com/gui-cs/Terminal.Gui/issues/3023.

I don't want to take on fixing that (#3023) in this PR, so I'm just temporarily commenting these out for now.